From 4f2182c86d34c80630fde5c86b56cb26cea1835c Mon Sep 17 00:00:00 2001
From: Marek Vasut <marex@denx.de>
Date: Fri, 5 May 2023 05:32:30 +0200
Subject: [PATCH 1/3] pixel_format: Fix ARGB8888 DRM format

The ARGB8888 pixel format should map to DRM format DRM_FORMAT_ARGB8888,
correct current typo which maps it to DRM_FORMAT_RGB565 .

Upstream-Status: Submitted [https://github.com/ardera/flutter-pi/pull/326]
Signed-off-by: Marek Vasut <marex@denx.de>
---
 include/pixel_format.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/pixel_format.h b/include/pixel_format.h
index 7314ceb..809cf5b 100644
--- a/include/pixel_format.h
+++ b/include/pixel_format.h
@@ -44,7 +44,7 @@ COMPILE_ASSERT(kMax_PixFmt == kRGBA8888_FpiPixelFormat);
 
 #define PIXFMT_LIST(V) \
     V( "RGB 5:6:5",    "RGB565",  kRGB565_FpiPixelFormat,   /*bpp*/ 16, /*opaque*/ true,  /*R*/ 5, 11, /*G*/ 6, 5,  /*B*/ 5, 0,  /*A*/ 0, 0,  /*GBM fourcc*/ GBM_FORMAT_RGB565,   /*DRM fourcc*/ DRM_FORMAT_RGB565) \
-    V("ARGB 8:8:8:8", "ARGB8888", kARGB8888_FpiPixelFormat, /*bpp*/ 32, /*opaque*/ false, /*R*/ 8, 16, /*G*/ 8, 8,  /*B*/ 8, 0,  /*A*/ 8, 24, /*GBM fourcc*/ GBM_FORMAT_ARGB8888, /*DRM fourcc*/ DRM_FORMAT_RGB565) \
+    V("ARGB 8:8:8:8", "ARGB8888", kARGB8888_FpiPixelFormat, /*bpp*/ 32, /*opaque*/ false, /*R*/ 8, 16, /*G*/ 8, 8,  /*B*/ 8, 0,  /*A*/ 8, 24, /*GBM fourcc*/ GBM_FORMAT_ARGB8888, /*DRM fourcc*/ DRM_FORMAT_ARGB8888) \
     V("XRGB 8:8:8:8", "XRGB8888", kXRGB8888_FpiPixelFormat, /*bpp*/ 32, /*opaque*/ true,  /*R*/ 8, 16, /*G*/ 8, 8,  /*B*/ 8, 0,  /*A*/ 0, 24, /*GBM fourcc*/ GBM_FORMAT_XRGB8888, /*DRM fourcc*/ DRM_FORMAT_XRGB8888) \
     V("BGRA 8:8:8:8", "BGRA8888", kBGRA8888_FpiPixelFormat, /*bpp*/ 32, /*opaque*/ false, /*R*/ 8,  8, /*G*/ 8, 16, /*B*/ 8, 24, /*A*/ 8, 0,  /*GBM fourcc*/ GBM_FORMAT_BGRA8888, /*DRM fourcc*/ DRM_FORMAT_BGRA8888) \
     V("RGBA 8:8:8:8", "RGBA8888", kRGBA8888_FpiPixelFormat, /*bpp*/ 32, /*opaque*/ false, /*R*/ 8, 24, /*G*/ 8, 16, /*B*/ 8, 8,  /*A*/ 8, 0,  /*GBM fourcc*/ GBM_FORMAT_RGBA8888, /*DRM fourcc*/ DRM_FORMAT_RGBA8888)
-- 
2.39.2

